services/proxy/pkg/user/backend/cs3.go

```diff
@@ -145,13 +145,25 @@ func (c *cs3backend) Authenticate(ctx context.Context, username string, password
 // user. If the user already exist this is not considered an error and the
 // function will just return the existing user.
 func (c *cs3backend) CreateUserFromClaims(ctx context.Context, claims map[string]interface{}) (*cs3.User, error) {
+	gatewayClient, err := c.gatewaySelector.Next()
+	if err != nil {
+		c.logger.Error().Err(err).Msg("could not select next gateway client")
+		return nil, err
+	}
 	newctx := context.Background()
-	token, err := c.autoProvsionCreator.GetAutoProvisionAdminToken(newctx)
+	authRes, err := gatewayClient.Authenticate(ctx, &gateway.AuthenticateRequest{
```

Comment:
Sure we want to use `ctx` not `newctx` here? `newctx` is used for all further actions

Reply:
Hm, true. Also I just noticed that I still need to adapt `ocis init` for the proxy's service account :disappointed:

Reply:
:heavy_check_mark: